### ISSUE TITLE
FedX throws error on ASK queries

### ISSUE THREAD

**Issue by mhoangvslev** [GDD-Nantes/FedShop#24]
# Diagnosis
- While debugging #21, I enabled `fedx:supportsASKQueries` in FedX config:

```ttl
<http://www.ratingsite2.fr/> a sd:Service ;
    fedx:store "SPARQLEndpoint";
    sd:endpoint "http://localhost:34209/sparql?default-graph-uri=http://www.ratingsite2.fr/";
    fedx:supportsASKQueries true . 
```

- During evaluation (`res.hasNext()`), FedX throws `"org.eclipse.rdf4j.query.QueryEvaluationException: @ sparql_www.ratingsite6.fr_ - HTTP Error: 406 Not Acceptable` on this subquery
```sparql
ASK WHERE  { <http://www.ratingsite6.fr/ProductFeature13619> <http://www.w3.org/2002/07/owl#sameAs> <http://www4.wiwiss.fu-berlin.de/bizer/bsbm/v01/vocabulary/ProductFeature12922> .  }
```
- Some digging on Virtuoso forums lead to this:
https://github.com/openlink/virtuoso-opensource/issues/464

# TL;DR
- RDF4J is tested on Virtuoso 6 and the bug appears in Virtuoso 7. Downgrade virtuoso ?
- Simple workaround:
```java
SPARQLRepository repo = new SPARQLRepository(host);
Map<String, String> headers =new HashMap<>();
headers.put("Accept", "application/sparql-results+json");
repo.setAdditionalHttpHeaders(headers);
```
